Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.9
```

Comment:
Probably could parameterize this along with the django version like we do in CI to ensure compatibility but I figure just getting something usable locally that is abstracted away from the host setup is better than nothing